lib/wakes/configuration.rb

```diff
@@ -15,9 +15,13 @@ def self.configure
 
   class Configuration
     attr_accessor :enabled
+    attr_accessor :ga_profiles
 
     def initialize
       @enabled = true
+      @ga_profiles = {
+        'default' => ENV['GOOGLE_ANALYTICS_PROFILE_ID']
```

Comment:
@benhutton It does make the configuration block easier to read, but it feels to me like a departure from how the rest of wakes handles hosts. Thinking about wakes as a published gem now: if you were to move your default domain AND you were using the `ga_profiles` setting, all of your wakes locations would continue redirecting and aggregating without modification, except for the `ga_profiles` setting. I liked that the 'default' behaviour was consistent in both places, i.e., there is always a fallback `default` host both for wakes locations and GA profiles. 

Maybe this is not worth worrying about, though, and I suppose one could always add 
`ENV['DEFAULT_HOST'] => ENV['GOOGLE_ANALYTICS_PROFILE_ID']` to the hash if you were worried about this.

Does that change anything in your mind?